cardinal/world.go

```diff
@@ -34,14 +34,24 @@ type World struct {
 	server          *server.Handler
 	evmServer       evm.Server
 	gameManager     *server.GameManager
-	isGameRunning   atomic.Bool
 	tickChannel     <-chan time.Time
 	tickDoneChannel chan<- uint64
 	serverOptions   []server.Option
 	cleanup         func()
-	endStartGame    chan bool
+
+	// gameSequenceStage describes what stage the game is in (e.g. starting, running, shut down, etc)
+	gameSequenceStage atomic.Int32
+	endStartGame      chan bool
 }
 
+const (
+	sequencePreStart int32 = iota
+	sequenceStarting
+	sequenceRunning
+	sequenceShuttingDown
+	sequenceShutDown
+)
```

Comment:
This is an enum, there should be a type alias for the atomic and the underlying int32 literals.

Reply:
I added a gamestage package to wrap the atomic struct with some concrete types.